t/02_attribute.t

```diff
@@ -0,0 +1,154 @@
+use Test::Most 0.22 (tests => 7);
+use Test::NoWarnings;
+
+use App::Config::Attribute;
+use Data::Hash::DotNotation;
+
+throws_ok {
+    App::Config::Attribute->new(
+        name        => 'test',
+        parent_path => 'apperturescience'
+    );
+}
+qr/Attribute \(data_set\) is required/;
+
+throws_ok {
+    App::Config::Attribute->new(
+        name        => 'suspend_trades',
+        parent_path => 'quants.underlyings',
```

Comment:
don't use "quants.underlyings". This is an open-source repo, which should have _nothing_ to do with anything Binary-related.